backend/src/clients/inferencing.ts

```diff
@@ -10,12 +10,12 @@ interface InferenceService {
   port: number
 }
 
-export async function createInferenceService(inferenceServiceParams: InferenceService) {
+export async function createInferenceService(inferenceServiceParams: InferenceService, token: string) {
```

Comment:
Instead of passing the `token` in, we could just get it from `config` in this file instead.